src/sections/tickets.tsx

```diff
@@ -0,0 +1,149 @@
+  >([]);
+  const [showStatistics, set_showStatistics] = useState(false);
+  const [showTickets, set_showTickets] = useState(false);
+
+  useEffect(() => {
+    if (loginData.apiEndpoint && loginData.apiToken) {
+      dispatch(
+        actionsAsync.getStatisticsThunk({
+          apiEndpoint: loginData.apiEndpoint,
+          apiToken: loginData.apiToken,
+        })
+      );
+      dispatch(
+        actionsAsync.getTicketsThunk({
+          apiEndpoint: loginData.apiEndpoint,
+          apiToken: loginData.apiToken,
+        })
+      );
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [loginData]);
+
+  return (
+    <Section className="Section--tickets" id="Section--tickets" yellow>
+      <h2>
+        Tickets{' '}
+        <button
+          onClick={() => {
+            if (loginData.apiEndpoint && loginData.apiToken) {
+              dispatch(
+                actionsAsync.getStatisticsThunk({
+                  apiEndpoint: loginData.apiEndpoint,
+                  apiToken: loginData.apiToken,
+                })
+              );
+              dispatch(
+                actionsAsync.getTicketsThunk({
+                  apiEndpoint: loginData.apiEndpoint,
+                  apiToken: loginData.apiToken,
+                })
+              );
+            }
+          }}
+        >
+          Refresh
+        </button>
+      </h2>
+      <div>Unredeemed: {statistics?.unredeemed}</div>
+      <div>Redeemed: {statistics?.redeemed}</div>
+      <RedeemTickets
+        onSuccess={() => {
+          set_redeemSuccess(true);
+        }}
+        onError={(e) => {
+          set_redeemSuccess(false);
+          set_redeemErrors([
+            ...redeemErrors,
+            {
+              error: e.error,
+              status: e.status,
+            },
+          ]);
+        }}
+      />
+      <div>
+        <button
+          onClick={() => {
+            set_showStatistics(!showStatistics);
+          }}
+        >
+          {showStatistics ? 'Hide Statistics' : 'Show Statistics'}
+        </button>
+        {showStatistics && (
+          <pre>
+            {statistics
+              ? JSON.stringify(statistics, null, 2)
+              : 'No statistics available'}
+          </pre>
+        )}
+      </div>
+      <div>
+        <button
+          onClick={() => {
+            set_showTickets(!showTickets);
+          }}
+        >
+          {showTickets ? 'Hide Tickets' : 'Show Tickets'}
+        </button>
+        {showTickets && (
+          <pre>
+            {tickets
+              ? JSON.stringify(tickets, null, 4)
+              : 'No tickets available'}
+          </pre>
+        )}
+      </div>
+    </Section>
+  );
+}
+
+function RedeemTickets({
+  onError,
+  onSuccess,
+}: {
+  onError: (e: typeof APIError.prototype) => void;
+  onSuccess: () => void;
+}) {
+  const dispatch = useAppDispatch();
+  const loginData = useAppSelector((selector) => selector.auth.loginData);
```

Comment:
Now that the hadle Functions are to be added such as `handleRefresh` or `handleRedeemAll`, there is no need have this component as it will take only a couple of lines in JSX.